nwbinspector/load_checks.py

```diff
@@ -0,0 +1,13 @@
+"""Authors: Cody Baker and Ben Dichter."""
+from copy import deepcopy
+
+from .utils import default_checks
+from .check_time_series import (
+    check_dataset_compression,
+    check_regular_timestamps,
+    check_data_orientation,
+)
+
+
+def load_checks(checks=deepcopy(default_checks)):
```

Comment:
why the deepcopy? Are you making changes to this dictionary?

Reply:
I guess not necessary; residual from the very first implementation of the loading which caused the dictionary to add duplicates of each check function each time the loading was called. Issue doesn't seem to be present anymore